components/browser/menu/src/main/res/layout/mozac_browser_menu_item_caption.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
With the new change, I think we are not going to need this.

Reply:
Sorry about that, went to lunch and hadn't finished up my changes. Should be ready for review now!

Reply:
Don't worry :)